Day2/BoxFactory.py

```diff
@@ -0,0 +1,30 @@
+class BoxFactory:
+    
+    packaging_dimensions = None
+    finished_packaging = []
+    
+    def __init__(self, packaging_dimensions):
+        self.packaging_dimensions = packaging_dimensions
+        self.finished_packaging = []
+
+    def make_box(self, length, width, height):
+        side1 = length*width
+        side2 = width*height
+        side3 = height*length
+        return (side1, side2, side3)
+    
+    def make_all_boxes(self):
+        for box in self.packaging_dimensions:
+            box_with_sides = self.make_box(*box)
+            self.finished_packaging.append(box_with_sides)
+    
+    def make_cylinder(self, radius, height):
+        return (radius, height)
+
+    def make_all_cylinders(self):
+        for cylinder in self.packaging_dimensions:
+            cylinder_with_sides_tops_bottoms = self.make_cylinder(*cylinder)
+            self.finished_packaging.append(cylinder_with_sides_tops_bottoms)
```

Comment:
igy h mind a kettot beleteszed a packaging nev az correct csak rossz classban van es amikor irtam a commentet eszembe nem jutott h lesz meg cylinder is a box classban

ezert is veszelyes ha rossz nevet adsz vminek mert nem lesz ideje kedve az embereknek az osszes classod elejetol a vegeig elolvasni ha nincs ra szukseguk

Reply:
egy szar kodbazisban 1 class lehet picsanagy, a capture onennal a legnagyobb class amit lattam 6000 soros volt. azt nem fogod vegigolvasni es ha vegig is olvasod nem fogod tudni fejben tartani
nyilvan a valtozokra azert ranezhetsz h mik hasznaljak stb 
de most a feladat h jo kodot irj :D nem h defensiven olvassam :D